sentry-kotlin-multiplatform/src/commonAppleMain/kotlin/io/sentry/kotlin/multiplatform/extensions/UserExtensions.kt

```diff
@@ -5,17 +5,16 @@ import io.sentry.kotlin.multiplatform.protocol.User
 
 internal fun User.toCocoaUser() = CocoaUser().apply {
     val scope = this@toCocoaUser
-    userId = scope.id
-    username = scope.username
-    email = scope.email
-    ipAddress = scope.ipAddress
+    this.userId = scope.id
```

Comment:
I don't think you need `this` when using `apply`, or is this because of the extension method?

Reply:
We don't need it but since other code parts were using it I wanted to keep things consistent

Unless you think it's better code practice to omit `this` when possible?

Reply:
One of the goals of scoped functions is to reduce boilerplate.
https://kotlinlang.org/docs/scope-functions.html
If you use eg `user.userId`, `user.email` the boiler plate is still there, from `user` to `this` is the same thing IMO.
You need to use `this` in case there's a shadowing issue tho, if it's not the case, I'd prefer to avoid `this`.